storage/cloudspanner/log_storage.go

```diff
@@ -296,17 +296,14 @@ func (ls *logStorage) AddSequencedLeaves(ctx context.Context, tree *trillian.Tre
 	errs := make(chan error, 1)
 	var wg sync.WaitGroup
 	for i, l := range leaves {
-		var err error
-		l.QueueTimestamp, err = ptypes.TimestampProto(ts)
-		if err != nil {
-			return nil, fmt.Errorf("got invalid queue timestamp: %v", err)
-		}
+		l.QueueTimestamp = timestamppb.New(ts)
```

Comment:
CheckValid here? and line 306?

Reply:
I can't find the thing on line 306. I think you're seeing an artifact of me moving the `err` definition down as it wasn't needed here any more. This one is checked now.